exchanges/kucoin/kucoin_wrapper.go

```diff
@@ -313,11 +313,18 @@ func (ku *Kucoin) UpdateTickers(ctx context.Context, assetType asset.Item) error
 		if err != nil {
 			return err
 		}
+		enabledPairs, err := ku.GetEnabledPairs(asset.Futures)
+		if err != nil {
+			return err
+		}
 		for x := range ticks {
 			pair, err := currency.NewPairFromString(ticks[x].Symbol)
 			if err != nil {
 				return err
 			}
```

Comment:
Use derive from method on an available pairs type.

Reply:
Anyways, I could use the enabled Pairs I get using **_ku.GetEnablePairs()_** method